### PR TITLE
Create trapping_rain_water.java

### PR DESCRIPTION
**Pull request by puneetsaluja100**
**Fixes issue:**
#5926

**Changes:**
Added trapping_rain_water.java

### CODE REVIEW

File: code/dynamic_programming/src/trapping_rain_water/trapping_rain_water.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    public static int trap_rain_water(int[] height) {
+        int result = 0;
+    
+        if(height==null)
+            return result;
+    
+        int left[] = new int[height.length];
+        int right[]= new int[height.length];
```
Comment (Devansh-Maurya) [inline code/dynamic_programming/src/trapping_rain_water/trapping_rain_water.java:14]: Please be consistent in formatting code.

**Review by Devansh-Maurya — CHANGES_REQUESTED** [review]

**Review by Devansh-Maurya — CHANGES_REQUESTED** [review]
Please follow the [Java Style Guide](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/java#expressions) while writing Java code. There are formatting issues at multiple places.

File: code/dynamic_programming/src/trapping_rain_water/trapping_rain_water.java (hunk 1 above)
Comment (puneetsaluja100) [reply code/dynamic_programming/src/trapping_rain_water/trapping_rain_water.java]: I have updated the changes suggested by you. Can you review it again.

**Review by AdiChat — APPROVED** [review]

**Comment by AdiChat** [comment]
Looks great 😍 

You have made a wonderful contribution. 
Join us [**at our community forum**](https://discourse.opengenus.org/signup) :family_man_woman_girl_boy: to get connected and access to community benefits

If you would like to [join our internship program](https://discourse.opengenus.org/t/internship-guidelines-at-opengenus/2335/) and contribute to [OpenGenus IQ](https://iq.opengenus.org/), let us know and we will be happy to have you onboard 

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus

**Pull request merged**